modules/cart/js/commerce_cart.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file
+ * Defines Javascript behaviors for the commerce cart module.
+ */
+
+(function ($, Drupal, drupalSettings) {
+  'use strict';
+
+  Drupal.behaviors.commerceCartBlock = {
+    attach: function (context) {
+      var $context = $(context);
+      var $cart = $context.find('.cart--cart-block');
+
+      $cart.hover(
```

Comment:
So I went browsing and it seemed everyone used hover. Clicking is counterintuitive in my opinion. I have to click cart, to open a dropdown, and then click a different cart link in order to view my cart. This supports clicking and going strait to cart